packages/providence/src/cli/generate-extend-docs-data.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const fs = require('fs');
+const pathLib = require('path');
```

Comment:
Bit confusing, would call it path to be conventional

Reply:
The reason I do this consistently throughout the whole package is that within Babel ast traversal, the `path` variable is used for the ast path